recipes/mbedtls/all/conanfile.py

```diff
@@ -108,6 +108,8 @@ def package_info(self):
 
         self.cpp_info.components["mbedcrypto"].set_property("cmake_target_name", "MbedTLS::mbedcrypto")
         self.cpp_info.components["mbedcrypto"].libs = ["mbedcrypto"]
+        if self.settings.os == "Windows":
+            self.cpp_info.components["mbedcrypto"].system_libs = ["bcrypt"]
```

Comment:
https://github.com/Mbed-TLS/mbedtls/blob/c3d17cde46415784b97e96ec3b7565bd4234ad5a/library/CMakeLists.txt#L219C22-L219C28 should `ws2_32` be added here too?

Reply:
@RubenRBS Yes, you are right. In my case ws2_32 was consumed via other dependencies, so I didn't notice that